### PR TITLE
Richard jan20 errata

### PR DESCRIPTION
**Pull request by madmongo1**
Corrects typos, grammar and links in richard's BIO and jan 2020 blog entry.

### CODE REVIEW

File: _posts/2020-01-31-RichardsJanuaryUpdate.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ I was personally impressed by the speed with which I was able to assimilate a ne
 service infrastructure using nothing more than boost, nlohmann_json, openssl and a c++17 compiler. In my view
 this would not have been possible without the excellent documentation and careful attention to detail in Boost.Beast.
 
-During the development, I reached out to Vinnie and Damien on slack a number of times. Both were always helpful 
+During the development, I reached out to Vinnie and Damien on Slack a number of times. Both were always helpful
```
Comment (glenfe) [inline _posts/2020-01-31-RichardsJanuaryUpdate.md]: Damian, not Damien.

File: _posts/2020-01-31-RichardsJanuaryUpdate.md [hunk 2]
```diff
@@ -128,8 +128,8 @@ you get time to do very often when working on an FX-options desk in an investmen
 
 I have been providing some support to the upcoming [Boost.JSON](https://github.com/vinniefalco/json) library.
 This library focusses on:
-* Absolute correctness in reference to (RFC8259)[https://datatracker.ietf.org/doc/rfc8259/]. 
-* Seeking to match or exceed the performance of other c++ JSON libraries such as (RapdJSON)[https://rapidjson.org/].
+* Absolute correctness in reference to [RFC8259](https://datatracker.ietf.org/doc/rfc8259/). 
+* Seeking to match or exceed the performance of other c++ JSON libraries such as [RapdJSON](https://rapidjson.org/).
```
Comment (glenfe) [inline _posts/2020-01-31-RichardsJanuaryUpdate.md]: RapidJSON, not RapdJSON.

**Comment by vinniefalco** [comment]
Needs squash

**Comment by madmongo1** [comment]
squashed

**Pull request merged**